tslint.json

```diff
@@ -85,8 +85,7 @@
       ["fit"],
       ["fdescribe"],
       ["xit"],
-      ["xdescribe"],
-      {"name": "Object.assign", "message": "Use the spread operator instead."}
```

Comment:
I don't think we need a new lint rule. We can just use the `ban` rule and fix the usage by using:

```ts
{"name": ["Object", "assign"], "message": "Use the spread operator instead."}
```

Reply:
Nice, that's even better. Changed.